ajax/libs/slickgrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "slickgrid",
+  "version": "2.3.4",
+  "description": "A lightning fast JavaScript grid/spreadsheet",
+  "filename": "slick.core.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/6pac/SlickGrid.git"
+  },
+  "keywords": [
+    "slickgrid",
+    "grid"
+  ],
+  "author": {
+    "name": "Michael Leibman",
+    "email": "michael.leibman@gmail.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/mleibman/SlickGrid/wiki",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/6pac/SlickGrid.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "lib/jquery.event.*",
```

Comment:
@hare1039 thank you! I find the `jquery.event.*` is from https://github.com/threedubmedia/jquery.threedubmedia . Ref: https://github.com/6pac/SlickGrid/blob/master/lib/jquery.event.drag-2.3.0.js#L4
https://github.com/6pac/SlickGrid/blob/master/lib/jquery.event.drop-2.3.0.js#L4

I'll add this library later.
And delete this line.